lib/Parser/perrors.h

```diff
@@ -96,7 +96,7 @@ LSC_ERROR_MSG(1084, ERRGetterMustHaveNoParameters, "Getter functions must have n
 
 LSC_ERROR_MSG(1085, ERRInvalidUseofExponentiationOperator, "Invalid unary operator on the left hand side of exponentiation (**) operator")
 
-LSC_ERROR_MSG(1086, ERRInvalidModuleImportOrExport, "Module import or export statement unexpected here")
+// 1086 -- removed LSC_ERROR_MSG(1086, ERRInvalidModuleImportOrExport, "Module import or export statement unexpected here")
```

Comment:
We should probably either remove the commented code completely, or reuse this error number (1086) for the  `ERRModuleImportOrExportInScript` error.

Reply:
There was a comment at t top of the file about not reusing error numbers and also further up there’s a commented out removed error so I was following what I perceived to be the pattern.

I figure I can reuse it as one of the two I’ve split it into though - will update later.

Reply:
Ah, you're right!

Let's reuse that one so we can avoid the cruft as much as possible. This is really great by the way!